Makefile

```diff
@@ -243,11 +244,13 @@ generate-crd:
 		output:dir='config/crd/bases' # config/crd/bases/{group}_{plural}.yaml
 
 generate-crd-docs:
+	install -d ./config/crd/generated
+	$(PGO_KUBE_CLIENT) kustomize ./config/crd > ./config/crd/generated/all.yaml
```

Comment:
❓ I think I'm going to remove these changes. The template we use now doesn't contain anything about validation, and that's all we get by running `kustomize` on this directory.

I dislike the `generated` directory that contains one file. I'd prefer to create a `generated.yaml` file that we commit and review during PRs (except the CRD file is already 400KB...) When https://github.com/fybrik/crdoc/commit/a3ed20a543e80 is released, we can reconsider what file(s) to commit, whether or not we want validation in the CRD docs, etc.

Reply:
Replaced with a TODO mentioning the CRD patches. bc27ad6524d55c4a8f3c993c1871064be9056a96